crates/sui-indexer/src/apis/governance_api_v2.rs

```diff
@@ -52,12 +261,13 @@ impl GovernanceReadApiServer for GovernanceReadApiV2 {
         &self,
         staked_sui_ids: Vec<ObjectID>,
     ) -> RpcResult<Vec<DelegatedStake>> {
-        // Need Dynamic field queries
-        unimplemented!()
+        self.get_stakes_by_ids(staked_sui_ids)
+            .await
+            .map_err(Into::into)
     }
 
     async fn get_stakes(&self, owner: SuiAddress) -> RpcResult<Vec<DelegatedStake>> {
-        // Need Dynamic field queries
+        // Need object_type column in objects table
```

Comment:
What about 
`select * from objects where df_object_type=’0x3::staking_pool::StakedSui’;`

Reply:
Thanks for adding the `get_stakes` function!